src/query/server/query_test.go

```diff
@@ -584,14 +569,14 @@ func TestNewPerQueryEnforcer(t *testing.T) {
 		"cost.reporter.over_datapoints_limit+enabled=true,limiter=global":  0,
 		"cost.reporter.datapoints_counter+limiter=global":                  11,
 		"cost.reporter.over_datapoints_limit+enabled=false,limiter=query":  0,
-		"cost.reporter.over_datapoints_limit+enabled=true,limiter=query":   1,
+		"cost.reporter.over_datapoints_limit+enabled=true,limiter=query":   0,
 	}
 	expectGaugeValues := map[string]float64{
-		"cost.limits.threshold+limiter=global":    100,
-		"cost.limits.enabled+limiter=global":      1,
+		"cost.limits.threshold+limiter=global":    0,
+		"cost.limits.enabled+limiter=global":      0,
```

Comment:
Should we expect to see these at all now?

Reply:
Removed